core/src/main/resources/error/error-classes.json

```diff
@@ -1009,6 +1009,11 @@
           "<value>."
         ]
       },
+      "REGEX_GROUP_INDEX_EXCEED_REGEX_GROUP_COUNT" : {
```

Comment:
Could you make it shorter, please. I think it is enough to just name the invalid parameter. For instance, `REGEX_GROUP_INDEX`.